README.md

```diff
@@ -18,13 +18,25 @@ The above example will output coverage metrics for `foo.js`.
 
 ## CLI Options / Configuration
 
-c8 can be configured via command-line flags, a `c8` section in `package.json`, or a JSON configuration file on disk.
-
-A configuration file can be specified by passing its path on the command line with `--config` or `-c`. If no config option is provided, c8 searches for files named `.c8rc`, `.c8rc.json`, `.nycrc`, or `.nycrc.json`, starting from
-`cwd` and walking up the filesystem tree.
+c8 can be configured via command-line flags, a `c8` section in `package.json`, or a configuration file on disk.
 
 When using `package.json` configuration or a dedicated configuration file, omit the `--` prefix from the long-form of the desired command-line option.
 
+A configuration file can be specified by passing its path on the command line with `--config` or `-c`. If no config option is provided, c8 searches for files named in the table below starting from `cwd` and walking up the filesystem tree.
+
+A variety of file names are available in an effort to stay backwards compatible with nyc and ensure dynamic configuration options depending on the build system.
```

Comment:
backward compatibility is good thing, but what is the preferred way for new users? I propose to add some note about 'modern' configuration system (c8.config.js, right?) and 'legacy' (and may be 'deprecated') options.

Reply:
I changed the wording to clarify what is supported for configuration options.  It includes currently supported ESM module options.